courier/template/courier/builtin/templates/recovery_code/valid/email.body.gotmpl

```diff
@@ -0,0 +1,5 @@
+Hi,
+
+please recover access to your account by entering the following code:
+
+{{ .RecoveryCode }}
```

Comment:
We could, but the purpose of the new strategy is that the user never leaves the window/tab in which they initiated the flow. 

Also, if a user opens the flow in a different browser, they will get a CSRF violation error and that is terrible UX. I'd leave it out to avoid these confusions.